components/NewNFTBook.vue

```diff
@@ -450,6 +452,16 @@ onMounted(async () => {
         if (classResData?.ownerWallet !== wallet.value) {
           throw new Error('NOT_OWNER_OF_NFT_CLASS')
         }
+        if (classResData.connectedWallets) {
+          const connectedWalletKeys = Object.keys(classResData.connectedWallets || {})
+          const firstConnectedWallet = connectedWalletKeys[0]
+
+          isUsingDefaultAccount.value = firstConnectedWallet === wallet.value
+          if (!isUsingDefaultAccount.value && firstConnectedWallet) {
+            stripeConnectWallet.value = firstConnectedWallet
+          }
+        }
```

Comment:
I think this block should be refactored into 1 `if-else` block together with line 434 to make this more readable
```
        try {
          await fetchStripeConnectStatusByWallet(wallet.value)
          if (getStripeConnectStatusByWallet.value(wallet.value).isReady) {
            isStripeConnectChecked.value = true
            stripeConnectWallet.value = wallet.value
          }
        } catch (err) {
          // eslint-disable-next-line no-console
          console.error(err)
        }
```